src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -673,3 +652,26 @@ def is_distributed(self) -> bool:
         if isinstance(self.accelerator, TPUAccelerator):
             is_distributed |= self.strategy.is_distributed
         return is_distributed
+
+
+def _set_torch_flags(*, deterministic: Optional[Union[bool, _LITERAL_WARN]] = None, benchmark: Optional[bool] = None):
```

Comment:
Only keyword arguments allowed to avoid potential bugs where the position does not match the argument